eli5/templates/weights.html

```diff
@@ -0,0 +1,71 @@
+{% for expl in explanations %}
+
+    <p>
+        <b>
+            {% if expl.class is defined %}
+                y={{ expl.class }}
+            {% elif expl.target is defined %}
+                {{ expl.target }}
+            {% endif %}
+        </b>
+        {% if expl.proba is defined or expl.score is defined %}
+            {% set comma = joiner(", ") %}
+            (
+            {%- if expl.proba is defined -%}
+                {{ comma() }}probability <b>{{ "%0.3f"|format(expl.proba) }}</b>
+            {%- endif -%}
+            {%- if expl.score is defined -%}
+                {{ comma() }}score <b>{{ "%0.3f"|format(expl.score) }}</b>
+            {%- endif -%}
+            )
+        {% endif %}
+        top features
+    </p>
+
+    {% if force_weights or not expl.weighted_spans is defined %}
+    <table class="eli5-weights">
+        <thead>
+        <tr>
+            <th>Feature</th>
+            <th>Weight</th>
+        </tr>
+        </thead>
+        <tbody>
+        {% with w = expl.feature_weights %}
+
+            {% for name, coef in w.pos %}
+                <tr>
+                    <td>{{ name|escape }}</td>
+                    <td>{{ "%+.3f"|format(coef) }}</td>
+                </tr>
+            {% endfor %}
+            {% if w.pos_remaining %}
+                <tr>
+                    <td>&hellip;</td>
+                    <td colspan="2">{{ w.pos_remaining }} more positive</td>
```

Comment:
@jnothman I don't know what should be a default, but an option and a toggler would be nice to have. Sorting coefficients by absolute value looks useful, and color allows to easily see if a feature is positive or negative, a good point.

We should definitely use more HTML features, not just display a table with coefficients.